worlds/witness/hints.py

```diff
@@ -202,8 +220,11 @@ def get_priority_hint_locations(multiworld: MultiWorld, player: int):
 
 
 def make_hint_from_item(multiworld: MultiWorld, player: int, item: str):
-    location_obj = multiworld.find_item(item, player).item.location
+    locations = multiworld.find_item_locations(item, player)
```

Comment:
I suppose the way to do this would be to keep a reference of everything I add to the multiworld.itempool?

This would prevent certain plandoed items from being hinted but. I mean I guess if someone plandoes something they already know where it is innit

Reply:
The get_locations(player) or get_location(name, player) functions are fine tho, right? If I recall correctly, I've been told that those are fast.

If that's the case, I think I have something better now.